index.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+   <link rel="stylesheet" href="style.css"> 
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+   <link href='http://fonts.googleapis.com/css?family=Lato:400,700' rel='stylesheet' type='text/css'>
+    <title>UI Design</title>
+  </head>
+  <body>
+    <header class="header"> 
+      <ul>
+         <li><a href="#" class="fa fa-twitter"></a> </li>
+        <li><a href="#" class="fa fa-facebook"></a> </li>
+        <li>English</li>
+        <li>My page</li>
+        <li>Layout</li>
+        
+      </ul>
+    </header>
+    <nav class="navbar-Mobile">
+      <div class="logonav">
+       <a href="about.html">
+        <img src="images/logo.avif" class="logonav" alt=""></a>
+      </div>
+     <ul class="menu">
+              <li><a href=""><img class="closebtn" src="images/AppBar.png" alt=""></a></li>
+       <li><a href="about.html">Home</a></li>
+       <li><a href="">program</a></li>
+       <li><a href="">News</a></li>
+       <li><a href="">Join</a></li>
+       <li><a href="">Sponsor</a></li>
+       <li><a href="">Compaign</a></li>
+     </ul>
+     <div class="hamburger">
+               <div class="bar"></div>
+       <div class="bar"></div>
+       <div class="bar"></div>
+       </div>
+    </nav>
+    <div class="wrapper">
+        
+      <div class="intro">
+  
+        <h3 class="title-home">"Hello Freshers!"</h3>
+        <h1 class="h1">Mobile first design strategy</h1>
+        <p class="info">
+          Mobile-first design is a key ingredient to successful product design.
+          Designing for the smallest screens first, and then working your way up
+          empowers designers to focus on the core functions of their product.
+          When you focus on the core of your product and strip away the rest,
+          you are able to pinpoint the most important UX components of the
+          product.
+        </p>
+        <p class="caption">
+          Discover some of the current trends for responsive user interface
+          design.
+        </p>
+        <a href="#" class="email">uiuxconference@gmail.com </a>
+      </div>
+      <div class="logosec">
```

Comment:
- [ ] `required`
- [ ] Kindly please make the design as close as possible to the original design 
- [ ] By Increasing the `font size` `width` and making sure everything is in line.

`provided design`
![image](https://user-images.githubusercontent.com/66230306/190280619-b8ec467f-8636-491d-b803-b72036257e29.png)

`your design`
![image](https://user-images.githubusercontent.com/66230306/190281204-6d68847f-d6eb-4e76-bf68-5f4569daa0a1.png)